public/pages/AnomalyCharts/containers/AnomalyHeatmapChart.tsx

```diff
@@ -193,14 +237,14 @@ export const AnomalyHeatmapChart = React.memo(
           );
           setHeatmapData([transparentHeatmapData, ...selectedHeatmapData]);
 
-          const selectedEndDate = moment(
+          const selectedStartDate = moment(
             //@ts-ignore
             heatmapData[0].x[selectedCellIndices[1]],
             HEATMAP_X_AXIS_DATE_FORMAT
           ).valueOf();
 
-          const selectedStartDate =
-            selectedEndDate -
+          const selectedEndDate =
+            selectedStartDate +
```

Comment:
Could you explain why the start and end dates are switched here?

Reply:
This is because I make [change](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/350/files#diff-cde386f024d306c06003870e2bde6d912b144586cbd972d40e77a00defddb529R273) switch the date of each cell in heatmap from endDate to startDate. I think showing the start date of heatmap cell as `Time` in the hover box is better.

Thus, the `x` of heatmap data here is changed to startDate, and I have to switch endDate and startDate here